### ISSUE TITLE
Quick switch recents does not work with CM13 Multi-window mode 

### ISSUE THREAD

**Issue by DerVerruckteFuchs** [wasdennnoch/AndroidN-ify#453]
Enabling multi-window mode in Cyanogenmod 13's developer settings prevents N-ify's double tap recents quick switch function from working. Instead the double tap takes you back to the current app as if the double tap recents function is disabled. Disabling Multi-window mode restores normal quick switch behavior.

**Comment by Sander9847** [comment]
Cm13 already has it's own way to switch to the last app I think. Try to enable it by your rom settings instead of the modules. I could set it as a double tap option to the home button.

**Comment by DerVerruckteFuchs** [comment]
I've used the double tap option on the home button for recent switching before, but it felt weird since I got used to double tapping recents for that on OxygenOS. Using the recents button became muscle memory for me after awhile and makes more sense since it's more app related and in Android N. I'd prefer it if I could use double tap on the recents button, as I've assigned double tapping on home to launching my camera app. For the moment I've disable multi-window, as quick switching is more important to me and I use it way more often. It would be nice to have both.

For whatever reason the CM13 devs thought it appropriate to remove double tap on recents due to UI design reasons. Something about it being messy for users, especially users unaware of the feature and getting confused then fumbling through recents. However, double tap options for recents and the back button exist.
